### ISSUE TITLE
Who owns digital cultural heritage?

### ISSUE THREAD

**Issue by valeriavitale** [fedenanni/HDS-DiscussionGroup#23]

**Comment by valeriavitale** [comment]
The debate around restitution and repatriation of cultural heritage acquired illegally and unethically by colonising countries is leaving out a critical reflection on digital reproductions of cultural heritage, as pointed out by [Wallace and Pavis in their response to the 2018 Sarr-Savoy report](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3378200). Who controls access to those digital objects? Who benefits from ditising and studying them? Who determines what should or should not be open access? How can AIs and other automated approaches comply with these kinds of ethical and legal issues?

**Comment by valeriavitale** [comment]
Possible discussion for after the Summer?

**Comment by fedenanni** [comment]
This is a really cool one! Do you have someone in mind we could invite?

**Comment by valeriavitale** [comment]
Yes, I'll try to invite Andrea Wallace, from Exeter. But she's super busy, so not sure she'll say yes ;-) She may suggest other people to get the conversation started. I also know a few colleagues in Classics who are very interested, and might like to join the discussion (but not lead it).
